app/views/content/blog/getting-ready-to-apply.md

```diff
@@ -67,24 +69,36 @@ An experienced teaching professional can help you if you're (one of the followin
 
 ## 5. Reflect on (or get some) school experience
 
-Demonstrating how any voluntary time you have spent in school has shaped your view of teaching is vital to a good application. If you haven’t managed to get some time in a school yet, some schools are offering the opportunity for candidates to undertake school experience both in school and virtually. You can search for schools and apply using our [Get School Experience](https://schoolexperience.education.gov.uk/) service.
+Demonstrating how any voluntary time you have spent in school has shaped your view of teaching is vital to a good application. If you haven’t managed to get some time in a school yet, some schools are offering the opportunity for candidates to undertake school experience both in school and virtually. You can search for schools and [apply for school experience](https://schoolexperience.education.gov.uk/).
```

Comment:
I appreciate that you've taken out the service name :-)

however, does this sentence quite make sense now? Should the link also contain "search for schools" or the sentence reworded slightly?